core_contracts/loans/loans/positions.py

```diff
@@ -189,26 +192,32 @@ def get_standing(self, _day: int = -1, _readonly: bool = False) -> dict:
 
         ratio = collateral * EXA // debt
 
-        if ratio > self._loans._mining_ratio.get() * EXA // POINTS:
-            if _day == -1 or _day == self._loans.getDay():
-                if _readonly:
-                    price = self.asset_db["bnUSD"].lastPriceInLoop()
+        if self._loans.getDay() < self._loans._continuous_reward_day.get():
+            if ratio > self._loans._mining_ratio.get() * EXA // POINTS:
+                if _day == -1 or _day == self._loans.getDay():
+                    if _readonly:
+                        price = self.asset_db["bnUSD"].lastPriceInLoop()
+                    else:
+                        price = self.asset_db["bnUSD"].priceInLoop()
                 else:
-                    price = self.asset_db["bnUSD"].priceInLoop()
-            else:
-                price = self.snaps_db[_day].prices["bnUSD"]
+                    price = self.snaps_db[_day].prices["bnUSD"]
 
-            bnUSD_debt: int = debt * EXA // price
-            if bnUSD_debt < self._loans._min_mining_debt.get():
+                bnUSD_debt: int = debt * EXA // price
+                if bnUSD_debt < self._loans._min_mining_debt.get():
+                    standing = Standing.NOT_MINING
```

Comment:
@Suyog007 - I think we were planning to remove the mining threshold. We can check with Scott, but I think we want give rewards all the way down to liquidation, in which case there is no need for the mining or not_mining states.

Reply:
Unless continuous rewards day is activated, we will have all the threshold on the loans and once the continuous rewards day is activated, we will have only mining and liquidation state i.e. all the users that are not on liquidation state will be on mining state.
https://github.com/balancednetwork/balanced-contracts/blob/continuous-integration/core_contracts/loans/loans/positions.py#L195